### PR TITLE
Fixed include error

### PR DESCRIPTION
**Pull request by maccesch**
Fixed an include error in the stereo module

### CODE REVIEW

**Comment by vpisarev** [comment]
it's not quite clear why some implementation-related headers have been moved from src to include subdirectory? If API is not exposed, the headers should not be put into include

**Comment by maccesch** [comment]
That was actually the problem. These aren't implementation specific but are included in stereo.hpp.

**Comment by vpisarev** [comment]
oops, sorry, I missed that

File: modules/stereo/include/opencv2/stereo.hpp [hunk 1]
```diff
@@ -47,8 +47,8 @@
 #include "opencv2/core.hpp"
 #include "opencv2/features2d.hpp"
 #include "opencv2/core/affine.hpp"
-#include "../../stereo/src/descriptor.hpp"
-#include "../../stereo/src/matching.hpp"
+#include "stereo/descriptor.hpp"
```
Comment (vpisarev) [inline modules/stereo/include/opencv2/stereo.hpp]: could you then replace "stereo/descriptor.hpp" with "opencv2/stereo/descriptor.hpp" and do the same with subsequent include too for better clarity?

**Comment by maccesch** [comment]
updated :)

**Comment by vpisarev** [comment]
thanks! :+1:

**Pull request merged**